boards/arm/96b_wistrio/96b_wistrio.dts

```diff
@@ -93,8 +94,8 @@
 	pinctrl-names = "default";
 	status = "okay";
 
-	lis3dh@32 {
-		compatible = "st,lis3dh";
+	lis3dh: lis3dh@32 {
+		compatible = "st,lis3dh", "st,lis2dh";
```

Comment:
why adding `"st,lis2dh"` here?

Reply:
LIS3DH uses LIS2DH driver.
Enabling LIS2DH driver need to set DT_HAS_ST_LIS2DH_ENABLED.
By adding "st,lis2dh" to the compatibile, this option turn on.

If only "st,lis3dh" exists in the compatible option, 
only enable DT_HAS_ST_LIS3DH_ENABLED, but DT_HAS_ST_LIS2DH_ENABLED is not enabled.